front/components/ViewFolderAPIModal.tsx

```diff
@@ -80,7 +80,9 @@ export function ViewFolderAPIModal({
     }
   };
 
+  const mediaQuery = window.matchMedia("(prefers-color-scheme: dark)");
   const theme = localStorage.getItem("theme");
+  const isDark = theme === "dark" || (theme === "system" && mediaQuery.matches);
```

Comment:
Great catch, thanks for taking the time to fix it yourself! 🙏🏻 
 
You should actually be able to reuse a hook that I've wrote and forgot to use here instead of those 3 lines: 
`const { isDark } = useTheme();`

Let me know if unclear 🙏🏻

Reply:
(Same for all other blocks 🙏🏻 )